src/components/MoneyRequestHeaderStatusBar.tsx

```diff
@@ -9,7 +9,7 @@ type MoneyRequestHeaderStatusBarProps = {
     icon: ReactNode;
 
     /** Banner Description */
-    description: string;
+    description: string | ReactElement;
```

Comment:
@allgandalf could you clarify what BE errors do you mean?
This component just renders the description display.
Since for the admin, the broken connection violation text can include TextLink (link to company cards), it returned as a component.
https://github.com/Expensify/App/blob/91e299d3eced64181ef7ade3572ca5508f77a270/src/components/MoneyRequestHeaderStatusBar.tsx#L18-L28